package.json

```diff
@@ -27,15 +30,16 @@
     "bcryptjs": "^2.4.3",
     "body-parser": "^1.19.0",
     "bootstrap": "^4.3.1",
+    "concurrently": "^4.1.2",
```

Comment:
what does this lib do?

Reply:
allows you to start server and client with one command in 1 terminal